pyproject.toml

```
@@ -41,8 +41,7 @@ dependencies = [
     'scipy >= 1.2.0',
     'mne >= 1.1',
     'pooch',
-    'torch',
-    'importlib-metadata; python_version<"3.8"',
+    'torch >= 2.0',
```

Great they finally released!
I don't think we need the (very restrictive) version pin however. A Python 3.11 install will use the 2.0 version while an older Python version will use whatever version (the most recent one with an available wheel).

```suggestion
    'torch',
```